resources/scss/componets/01_basics.scss

```diff
@@ -0,0 +1,173 @@
+/*Rubic font*/
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Rubik:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
```

Comment:
Move this to **resources/views/layouts/components/assets.blade.php**

Reply:
Fixed